exchanges/okx/okx_wrapper.go

```diff
@@ -0,0 +1,1327 @@
+// submitted
+func (ok *Okx) WithdrawFiatFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is submitted
+func (ok *Okx) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (ok *Okx) GetActiveOrders(ctx context.Context, req *order.GetOrdersRequest) (order.FilteredOrders, error) {
+	err := req.Validate()
+	if err != nil {
+		return nil, err
+	}
+	if !ok.SupportsAsset(req.AssetType) {
+		return nil, fmt.Errorf("%w: %v", asset.ErrNotSupported, req.AssetType)
+	}
+	instrumentType := ok.GetInstrumentTypeFromAssetItem(req.AssetType)
+	var orderType string
+	if req.Type != order.UnknownType {
+		orderType, err = ok.OrderTypeString(req.Type)
+		if err != nil {
+			return nil, err
+		}
+	}
+	requestParam := &OrderListRequestParams{
+		OrderType:      orderType,
+		After:          req.StartTime,
+		Before:         req.EndTime,
+		InstrumentType: instrumentType,
+	}
+	response, err := ok.GetOrderList(ctx, requestParam)
+	if err != nil {
+		return nil, err
+	}
+	orders := make([]order.Detail, len(response))
+	for x := range response {
+		orderSide := response[x].Side
+		var pair currency.Pair
+		pair, err = ok.GetPairFromInstrumentID(response[x].InstrumentID)
+		if err != nil {
+			return nil, err
+		}
+		for i := range req.Pairs {
+			if !req.Pairs[i].Equal(pair) {
+				continue
+			}
+			orderStatus, err := order.StringToOrderStatus(strings.ToUpper(response[x].State))
+			if err != nil {
+				log.Errorf(log.ExchangeSys, "%s %v", ok.Name, err)
+			}
+			var oType order.Type
+			oType, err = ok.OrderTypeFromString(response[x].OrderType)
+			if err != nil {
+				return nil, err
+			}
+			orders[x] = order.Detail{
+				Amount:          response[x].Size,
+				Pair:            pair,
+				Price:           response[x].Price,
+				ExecutedAmount:  response[x].FillSize,
+				RemainingAmount: response[x].Size - response[x].FillSize,
+				Fee:             response[i].TransactionFee,
+				FeeAsset:        currency.NewCode(response[x].FeeCurrency),
+				Exchange:        ok.Name,
+				OrderID:         response[x].OrderID,
+				ClientOrderID:   response[x].ClientSupplierOrderID,
+				Type:            oType,
+				Side:            orderSide,
+				Status:          orderStatus,
+				AssetType:       req.AssetType,
+				Date:            response[x].CreationTime,
+				LastUpdated:     response[x].UpdateTime,
+			}
+		}
+	}
+	order.FilterOrdersByPairs(&orders, req.Pairs)
+	order.FilterOrdersByType(&orders, req.Type)
+	order.FilterOrdersBySide(&orders, req.Side)
+	return orders, order.FilterOrdersByTimeRange(&orders, req.StartTime, req.EndTime)
```

Comment:
This has been changed in a recent PR, please review other exchange's implementations of filtering and use that